src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -38,6 +42,8 @@ function BaseSidebarScreen(props) {
                         <SubNavigation
                             onLinkClick={startTimer}
                             insets={insets}
+                            // Don't display radius if platform is desktop. In this cases HeaderGap component is handling it.
+                            shouldDisplayRadius={!isDesktop}
```

Comment:
Why does this only affect desktop and not web?

Reply:
In desktop we want the radius to 'hug' the top of the window and at the same time we don't want clickable components to be overlapped by the top bar of the window. To achieve it we used the `HeaderGap` component which is rendered above the `Sidebar` screen component. If we won't disable the radius here we would end up with two radius one over the another.
On other platforms, there is no need to use the `HeaderGap` to achieve this. Using the `HeaderGap` solution on them as well would end up with excessive padding on the top of the sidebar.